src/utilities/effects.ts

```diff
@@ -3714,10 +3715,12 @@ const effects = {
       if (!queryPermissions.UPDATE_ACTIVITY_PRESET(user, updatedActivityPreset)) {
         throwPermissionError('update this activity preset');
       }
-
+      const { id: _id, ...restOfPresetPayload } = updatedActivityPreset;
```

Comment:
I believe this is a bug and should be `id: id` based on the method signature. Specifically, I believe this is causing [this bug](https://github.com/NASA-AMMOS/aerie-ui/issues/1018)

Reply:
@Mythicaeda that actually just removes `id` from  `updatedActivityPreset`. It's a way to delete `id` from the object using [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) without actually doing `delete updatedActivityPreset.id`, which will cause the original object to be mutated.

Effectively, this prevents `id` from being sent in the payload, which is `restOfPresetPayload`

Reply:
Should `id` actually be sent in an update?

Reply:
Yeah, I realized while creating #1020 to fix this that I had the right method, but wrong line. 

Although this insight makes me wonder if I should revert the change to the method signature I made there.

Reply:
I just saw your PR, and I think it's a reasonable change. I'll comment more on #1020